content/en/agent/faq/windows-agent-ddagent-user.md

```diff
@@ -79,11 +79,13 @@ If you’re using the Attach API, the change in user context means that the Agen
 
 ### Process check
 
-Now that the Agent runs under `ddagentuser`, it does not have access to the full command line of processes running under other users and to the user of other users’ processes. This causes the following options of the check to not work:
+The Agent since v6.11 runs as `ddagentuser` instead of `Local System`. Because of this, it does not have access to the full command line of processes running under other users and to the user of other users’ processes. This causes the following options of the check to not work:
```

Comment:
```suggestion
In v6.11 +, the Agent runs as `ddagentuser` instead of `Local System`. Because of this, it does not have access to the full command line of processes running under other users and to the user of other users’ processes. This causes the following options of the check to not work:
```